pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -552,11 +552,18 @@ def select_precision_plugin(self) -> PrecisionPlugin:
                 return TPUHalfPrecisionPlugin()
 
             if self.amp_type == AMPType.NATIVE:
+                if self.amp_level is not None:
+                    raise MisconfigurationException(
```

Comment:
I would just make it a a warning

Reply:
it's now None by default. if someone sets it explicitly, we should expect them to set this correctly as well otherwise an exception seems ok to me.. also it happens during init so one doesn't have to wait to realize the mistake later at some point in case they ignore the warning.